src/olympia/discovery/views.py

```diff
@@ -28,54 +29,53 @@ def get_params(self):
             params['locale'] = lang
         return params
 
-    def get_discopane_items(self):
-        if not getattr(self, 'discopane_items', None):
-            params = self.get_params()
-            edition = params.pop('edition', 'default')
-            self.discopane_items = discopane_items.get(
-                edition, discopane_items['default'])
-            if edition == 'china':
-                # No TAAR for China Edition.
-                telemetry_id = None
-            else:
-                telemetry_id = params.pop('telemetry-client-id', None)
-            if switch_is_active('disco-recommendations') and telemetry_id:
-                recommendations = get_recommendations(
-                    telemetry_id, params)
-                if recommendations:
-                    # if we got some recommendations then replace the
-                    # extensions in discopane_items with them.
-                    # Leave the non-extensions (personas) alone.
-                    self.discopane_items = replace_extensions(
-                        self.discopane_items, recommendations)
-            if switch_is_active('disco-staticthemes-dev'):
```

Comment:
Frontend devs need to be able to develop for static themes in the discopane, so you can't remove this code yet, unless we're including at least one static theme in the editorial data.   If that's the case can you comment in #8572 so Rebecca knows it's changed.  (Also, that code lasted a whole 14 days on -dev 😭 )

Reply:
My plan was to add a static theme in dev's data (since it's separate from prod/stage, being stored in the database), with a `position` set.